docker/Dockerfile

```diff
@@ -49,12 +49,26 @@ RUN mkdir /env && \
     rm -rv /env.tar.gz && \
     /env/bin/python /env/bin/conda-unpack && \
     useradd -u 8877 ghsci && \
-    # install tools for using OpenStreetMap and constructing uharfbuzz for text shaping in pdfs
-    apt-get update && apt-get install -y gcc g++ git osm2pgsql osmctools
+    # Install tools for using OpenStreetMap and constructing uharfbuzz for text shaping in pdfs
+    apt-get update && apt-get install -y gcc g++ git osm2pgsql osmctools && \
+    # Install the Google Cloud SDK
+    apt-get update && apt-get install -y apt-transport-https ca-certificates gnupg curl && \
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | gpg --dearmor -o /usr/share/keyrings/cloud.google.gpg && \
+    echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
+    apt-get update && apt-get install -y google-cloud-cli
 
-ENV PATH="/env/bin/:$PATH"
+ENV PATH="/env/bin/:$PATH"  
 ENV PROJ_LIB="/env/share/proj"
 
+# Install additional Python packages including earthengine-api and geemap
+RUN pip install uharfbuzz && \
```

Comment:
Looks like you are using an older version of the Dockerfile here --- have you merged the latest changes in?  This contains some old things we don't have any more (see https://github.com/healthysustainablecities/global-indicators/blob/main/docker/Dockerfile --- its more streamlined)